### PR TITLE
SUBMARINE-724. Reorganize Submarine Documentation

### PR DESCRIPTION
**Pull request by pingsutw**
### What is this PR for?

- Sort documents into three categories (Admin, user, developer)
- Add `getting started` docs for beginning users
- Update the documents, so that we can deploy them by using https://v2.docusaurus.io/docs/installation 
- Remove some outdated and empty documents
- move ` docs/database` to `dev-support`. (I think those files are for developers)

Preview the website: https://pingsutw.github.io/docs/ 

### What type of PR is it?
[Improvement  | Documentation]

### Todos
* [ ] - Update submarine-site

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-723

### How should this be tested?
https://travis-ci.com/github/pingsutw/hadoop-submarine/builds/215412351

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by tangzhankun** [comment]
@pingsutw Thanks a lot for the contribution! It is a huge improvement for our docs.
One question, for the deployment of the new docs, do we need to update https://github.com/apache/submarine-site?
I'm not very clear of the new process of updating the docs.

**Comment by pingsutw** [comment]
@tangzhankun, I've migrated all the contents include the download page in [old website](https://submarine.apache.org/) to https://pingsutw.github.io/ 

For the deployment process, we could add a Github Action in this repo to automatically deploy the website when someone updates the docs. Related to https://v2.docusaurus.io/docs/2.0.0-alpha.69/deployment#deploy

For the developer, they only need to update the docs in the `submarine` repo, and they don't need to care about the `submarine-site` repo, Github Actions will help them deploy the website in the future. Thus, the `submarine-site` only store HTML, css, and js files, and we don't need to maintain that repo anymore.

For the local deployment, we could use `yarn start` to starts a local development server. More details in https://github.com/pingsutw/hadoop-submarine/tree/SUBMARINE-724/website

I temporarily deploy the docs to my GitHub website (https://github.com/pingsutw/pingsutw.github.io) using `yarn deploy`. 
If there is no any problem, I will deploy the docs to `submarine-site`.

cc @xunliu @wangdatan @jiwq

File: pom.xml [hunk 1]
```diff
@@ -566,7 +566,9 @@
             <exclude>NOTICE-binary</exclude>
             <exclude>**/go.mod</exclude>
             <exclude>**/go.sum</exclude>
-            <exclude>**/*.ipynb</exclude>
+	    <exclude>**/*.ipynb</exclude>
+	    <exclude>**/.nojekyll</exclude>
+	    <exclude>**/yarn.lock</exclude>
```
Comment (xunliu) [inline pom.xml:571]: Please keep the same indentation.

File: website/docs/community/contributing.md [hunk 2]
```diff
@@ -12,8 +16,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Contribution Guidelines
-
 **Apache Hadoop Submarine** is an [Apache 2.0 License](https://github.com/apache/submarine/blob/master/LICENSE) Software.
```
Comment (xunliu) [inline website/docs/community/contributing.md:19]: Need changed `Apache Hadoop Submarine` to `Apache Submarine`.

File: website/docs/community/contributing.md [hunk 3]
```diff
@@ -12,8 +16,6 @@ See the License for the specific language governing permissions and
...
 
 Contributing to Hadoop Submarine (Source code, Documents, Image, Website) means you agree to the Apache 2.0 License.
```
Comment (xunliu) [inline website/docs/community/contributing.md:21]: Need change `Hadoop Submarine` to `Apache Submarine`.

**Review by xunliu — COMMENTED** [review]
@pingsutw Thank you for contribution. :-)

**Comment by tangzhankun** [comment]
@pingsutw That's a great process for updating our docs. Thanks! This LGTM. +1.

File: website/docs/community/contributing.md (hunk 3 above)
Comment (pingsutw) [reply website/docs/community/contributing.md]: Done

File: website/docs/community/contributing.md (hunk 2 above)
Comment (pingsutw) [reply website/docs/community/contributing.md]: Done

File: pom.xml (hunk 1 above)
Comment (pingsutw) [reply pom.xml]: Updated it.

**Pull request closed without merging**